### ISSUE TITLE
Introduce org-roam-weeklies

### ISSUE THREAD

**Issue by Abdisalan** [org-roam/org-roam#2283]
### Brief Abstract
Introduce feature similar to org-roam-dailies but handled at the weekly level.

### Long Description
org-roam-dailies is a feature that manages your day to day notes. Sometimes though, tasks may be unfinished at the end of the day and need to moved to the next day. Or you might want to plan a week long effort that would be difficult to coordinate between different daily notes.

org-roam-weeklies would let you manage your notes at the weekly level. Titles would adhere to the ISO 8601 date and time standard (week numbering and weeks that start on Monday). For example, this week's title could be "W47: 2022-11-21 to 2022-11-27"

### Please check the following:

- [x] No similar feature requests

**Comment by nickanderson** [comment]
On the surface it seems like something that could be implemented with specific templates with the daily templates.

I tried and failed with the following:


```
  (defun iso-title(year week)
    ;; Adapted from https://stackoverflow.com/a/51719844/2854727
    (concat
     (format-time-string "#+title: WW%V (%A %B %d - " (iso-beginning-of-week year week))
     (format-time-string "%A %B %d, %Y)" (iso-end-of-week year week))))

  (setq org-roam-dailies-capture-templates
        `(
          ("w" "Weekly" entry
           "* %?"
           :if-new (file+head ,(expand-file-name "weekly/%<%Y-W%W>.org" org-roam-dailies-directory)
                              (iso-title "%<%Y>" "%<%W>"))
           :clock-in nil
           :clock-resume nil
           )
          )
        )
```

**Comment by gamb** [comment]
Duplicate of https://github.com/org-roam/org-roam/issues/1411 ?

**Comment by nickanderson** [comment]
> Duplicate of https://github.com/org-roam/org-roam/issues/1411 ?

Yeah, pretty similar.

**Comment by Abdisalan** [comment]
Looks like this is resolved by #2010 by using a capture template
